deploy/crds/push_v1alpha1_androidvariant_cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: push.aerogear.org/v1alpha1
+kind: AndroidVariant
+metadata:
+  name: example-androidvariant
+spec:
+  # Add fields here
+  variantId: 'deadbeef-badf00d'
```

Comment:
is this field generated by UPS, or do we expect user to specify the id? If it is generated, then it shouldn't be in the `spec` field, but in `status` field instead as data in `spec` fields are normally entered by users, not generated.

Reply:
yeah, see the TODO in the PR description. Doing that change currently.